packages/plugins/FileService/src/schema-v2.json

```diff
@@ -40,6 +40,14 @@
         "landingTxID": {
             "type": "string",
             "title": "landingTxID"
+        },
+        "meson": {
```

Comment:
why not `useCDN`?

Reply:
It comes from a convention with the mobile team.